### PR TITLE
Upgrade to Django 1.10

### PR DESCRIPTION
**Pull request by frankh**
Finally up to date! (Until 1.11 is released next month)

### CODE REVIEW

File: cabot/settings.py [hunk 1]
```diff
@@ -241,7 +241,7 @@
         # Catch All Logger -- Captures any other logging
         '': {
             'handlers': ['console', 'log_file', 'mail_admins'],
-            'level': 'INFO',
+            'level': 'WARNING',
```
Comment (JeanFred) [inline cabot/settings.py]: Any reason for that change ?

File: requirements.txt [hunk 2]
```diff
@@ -27,7 +26,9 @@ PyJWT==0.1.2
 pyOpenSSL==16.1.0
 python-dateutil==2.1
 pytz==2014.10
+redis
```
Comment (JeanFred) [inline requirements.txt]: No pinnigng?

File: requirements.txt [hunk 3]
```diff
@@ -27,7 +26,9 @@ PyJWT==0.1.2
...
 requests==2.7.0
 twilio==3.4.1
+vine==1.1.3
```
Comment (JeanFred) [inline requirements.txt]: Can you drop a line in the commit message on why the additional requirement?

File: requirements.txt (hunk 2 above)
Comment (frankh) [inline requirements.txt]: I'm planning on rewriting the requirements using Pipenv to manage explicit vs implicit pinning

File: requirements.txt (hunk 3 above)
Comment (frankh) [inline requirements.txt]: I'm hoping I can solve this, seems to be a bug in celery/django 1.10

File: cabot/settings.py (hunk 1 above)
Comment (frankh) [inline cabot/settings.py]: The tests started spamming loads of log messages from celery... not sure why

File: requirements.txt (hunk 2 above)
Comment (frankh) [inline requirements.txt]: @JeanFred i've created a PR for updating requirements using pipenv

File: requirements.txt (hunk 3 above)
Comment (frankh) [inline requirements.txt]: Solved in update-requirements PR

**Comment by JeanFred** [comment]
LGTM!

**Pull request merged**